test/odbc/odbc_handler.h

```diff
@@ -108,6 +108,8 @@ class OdbcHandler {
 
     void BindColumns(vector<tuple<int, int, SQLPOINTER, int>> columns);
 
+    void BindColumns(vector<tuple<int, int, SQLPOINTER, int, SQLLEN* >> columns);
```

Comment:
Could we add a comment explaining what this `BindColumns` function does? How does this differ from the `BindColumns` function on line 109?